### DISCUSSION TITLE
PR conversation resolution

### DISCUSSION THREAD

**Discussion by acdvs** [community/community#3654] [Mobile]
Since PRs can get extremely cluttered with review comments/conversations, sometimes it's very difficult to scroll through because resolved conversations don't show as collapsed on mobile.

1. Collapse resolved conversations on mobile
2. Add buttons to resolve/unresolve conversations

**Comment by aconsuegra** [comment]
Thanks for your feedback @acdvs!

Can you share in which platform are you using the app?

**Reply by acdvs** [reply]
iOS 14.2
